mozilla/foundation.mozilla.org
# breadcrumb component for research hub pages

Closes #8498 
Related PRs/issues #


This PR introduces a new template for the research hub breadcrumbs component, and then adds that component to every research hub page. 

The template does not have much in terms of x/y spacing, as this differs between each page in the research hub. 
My thinking being that when we do the styling tickets of the pages, we can simply wrap this component in tailwind divs of our desired spacing. 

**Steps to test**:
1. Check out this branch on your local machine
2. visit localhost:8000/research, check that the title looks OK.
3. Visit localhost:8000/research/library/, and the title should now have breadcrumbs
4. Click on a research detail link, and the breadcrumbs should still render correctly.
5. Visit the authors/author detail page and check the breadcrumbs.
6. If everything is appearing as expected, testing is complete!
